src/scripts/oakItems/OakItems.ts

```diff
@@ -103,16 +98,6 @@ class OakItems implements Feature {
                 this.itemList[OakItems.OakItem[oakItem]].fromJSON(json[oakItem]);
             }
         });
-
-        // Loading purchaseList for BoughtOakItems
-        const purchaseList = json['purchaseList'];
-        if (purchaseList == null) {
-            this.purchaseList = this.defaults.purchaseList.map((v) => ko.observable<boolean>(v));
-        } else {
-            (purchaseList as boolean[]).forEach((value: boolean, index: number) => {
-                this.purchaseList[index](value);
-            });
-        }
```

Comment:
With `0.6.0` just being published, I think we will need to add something to `Updates.ts` for this?

Reply:
Just added one for 0.6.1